WcaOnRails/app/views/competitions/_competition_info.html.erb

```diff
@@ -39,7 +39,7 @@
         <dd><%=md competition.contact %></dd>
       <% end %>
 
-      <% if competition.base_entry_fee %>
+      <% if competition.base_entry_fee > 0 %>
```

Comment:
I'd really prefer a `.has_entry_fee?` function.

 I don't care if it uses `.non_zero` or `> 0` internally.